Black-Market/Black-Market/Components/MarketTexfield.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  MarketTexfield.swift
+//  Black-Market
+//
+//  Created by Junior Sancho on 7/12/23.
+//
+
+import SwiftUI
+
+struct MarketTexfield: TextFieldStyle {
+    @Binding var style: Style
+    private let title: String?
+        
+    enum Style {
+        case password
+        case `default`
+        case search
+        
+        var icon: Image? {
+            switch self {
+            case .password:
+                return Image("visibilityOffIcon")
```

Comment:
As I mentioned before, we can apply the same with the images

```
import SwiftUI

extension Image {
   static let search = Image("searchIcon")
   static let visibilityOff = Image("visibilityOffIcon")
}
```

Reply:
Done